admin/namespace.go

```diff
@@ -0,0 +1,62 @@
+package admin
```

Comment:
See earlier comment about `/api/namespace`, etc. You won't need separate `admin/namespace.go` and `admin/buckets.go` files.